### PR TITLE
Newspaper archive benefit and new New benefit pill

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What are you doing in this PR?

Adding an AB test for testing the newspaper archive benefit on the checkout landing page. Including a new "New" benefit pill. This constitutes Variant A. Variant B still to come (banner with images).

[**Trello Card**](https://trello.com/c/gFLnj2Vu/1017-add-newspaper-archive-benefit-to-lp-new-benefit-tooltip-and-checkout-test-a-b-c)

## Why are you doing this?

Testing the new newspaper archive benefit at point of conversion.

## How to test

(Turn on AB test) go to `uk/contribute#ab-newspaperArchiveBenefit=v1` and see "New" benefit

## Screenshots
image

### CODE REVIEW

File: support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx [hunk 1]
```diff
@@ -57,18 +58,19 @@ const tableCss = (style: CheckListStyle) => css`
...
 	strong?: boolean;
+	isNew?: boolean;
 };
 
 type CheckListStyle = 'standard' | 'compact' | 'hidden';
 
-export type CheckListProps = {
-	checkListData: CheckListData[];
+export type BenefitsCheckListProps = {
+	checkListData: BenefitsCheckListData[];
```
Comment (jamesgorrie) [inline support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx:73]: It's a little one, but having the prop and the type with different names can be a bit confusing.

File: support-frontend/assets/components/checkoutBenefits/newBenefitPill.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	background-color: ${news[400]};
+	color: ${neutral[100]};
+	${textSansBold14};
+	border-radius: 4px;
+	padding: 0 ${space[1]}px;
+`;
+
+export function NewBenefitPill() {
+	return <span css={newBenefitPill}>New</span>;
+}
```
Comment (jamesgorrie) [inline support-frontend/assets/components/checkoutBenefits/newBenefitPill.tsx:19]: If we don't use this anywhere outside of the benefits, should we keep it in that component?

File: support-frontend/assets/helpers/productCatalog.tsx [hunk 3]
```diff
@@ -312,3 +312,33 @@ export const productCatalogDescription: Record<ProductKey, ProductDescription> =
...
+			{
+				copy: 'Unlimited access to the Guardian Feast App',
+				isNew: true,
+				tooltip:
+					'Make a feast out of anything with the Guardian’s new recipe app. Feast has thousands of recipes including quick and budget-friendly weeknight dinners, and showstopping weekend dishes – plus smart app features to make mealtimes inspiring.',
+			},
+		],
+		offers: [],
+	},
+};
```
Comment (jamesgorrie) [inline support-frontend/assets/helpers/productCatalog.tsx:344]: I know this is a pattern we've had before, but I've found it a little cumbersome. Maybe we could just add it to the catalog with a `shouldRender` method that takes the switches and whatever data to calculate that?

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx [hunk 4]
```diff
@@ -437,6 +438,12 @@ export function ThreeTierLanding({
 	const selectedContributionRatePlan =
 		contributionType === 'ANNUAL' ? 'Annual' : 'Monthly';
 
+	const productCatalogDescription = ['v1', 'v2'].includes(
+		abParticipations.newspaperArchiveBenefit,
+	)
+		? productCatalogDescriptionNewBenefits
+		: canonicalProductCatalogDescription;
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:445]: Why'd we have 2 variants if we do the same for them both?

**Review by jamesgorrie — APPROVED** [review]
Couple of post-PR questions.

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx (hunk 4 above)
Comment (jamesgorrie) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:445]: > Variant B still to come (banner with images).

Got it

File: support-frontend/assets/components/checkoutBenefits/newBenefitPill.tsx (hunk 2 above)
Comment (andrewHEguardian) [reply support-frontend/assets/components/checkoutBenefits/newBenefitPill.tsx:19]: I think it's part of the design of the product banner
image

**Pull request merged**